mix.exs

```diff
@@ -26,7 +26,8 @@ defmodule Closex.Mixfile do
   end
 
   def application do
-    [mod: {Closex.Application, []}]
+    [mod: {Closex.Application, []},
+     env: [mock_client_fixtures_dir: "test/fixtures"]]
```

Comment:
Why is this set here?

Reply:
Ignore this, I'm trying to debug an issue when using this change